docs/react-web-core/pre-call/3-virtual-bg.mdx

```diff
@@ -37,9 +37,22 @@ import DyteVideoBackgroundTransformer from '@dytesdk/video-background-transforme
 If you installed the package using a script tag, the `DyteVideoBackgroundTransformer` will be automatically available for use once the installation process is complete.
 
 ```js
-const videoBackgroundTransformer = await DyteVideoBackgroundTransformer.init();
+const videoBackgroundTransformer = await DyteVideoBackgroundTransformer.init({
+            meeting,
+            segmentationConfig: {
```

Comment:
segmentationConfig is optional. They don't have to pass it. I exposed it for our usage.